1080i/Includes_Dimensions.xml

```diff
@@ -99,6 +99,13 @@
         <centertop>dialog_tall_centertop</centertop>
     </include>
 
+    <include name="Dimension_DialogSelectGame">
+        <right>grid_x04</right>
+        <width>680</width>
+        <height>980</height>
+        <centertop>50%</centertop>
```

Comment:
instead of centertop use top to show settings/clock icon in top right and size the dialog appropriately to the list size
(120 * 6 items + pad + pad) = 800px
40 is dialog pad value, items are 120 high, so we get six items on 1080p once we leave enough space for the header

```
<top>160</top>
<height>800</height>
```